.github/workflows/publish-barshare-gbfs.yml

```diff
@@ -0,0 +1,49 @@
+name: build & deploy publish-barshare-gbfs Docker image
+
+on:
+  push:
+    branches:
+      - main
+      # todo: remove
+      - ci-publish-bbnavi
```

Comment:
This is only for demonstration purposes as long this branch is not merged, I suppose?
Apparently some credentials/grants are still missing, as the build failed:
```
Error: buildx failed with: error: denied: requested access to the resource is denied
```

Reply:
Yes, this is just in place to make sure that the deployment actually works before merging into `main`. Let's keep this thread unresolved until we removed the `ci-publish-bbnavi` branch exception.